ci/env.sh

```diff
@@ -0,0 +1,21 @@
+#smoke test environment specifications:
+
+#for scale perf
+export SCALE=4
+export RUNS=1
+
+#for kube burner 
+export JOB_ITERATIONS=3
```

Comment:
Do we need 3 iterations for the test?

Reply:
We define the number of projects to create using the JOB_ITERATIONS var.